### PR TITLE
Dashboard page

### PR DESCRIPTION
**Pull request by Nadeen123**
#11
created a dashboard page in iPhone SE
created dashboard styled component

### CODE REVIEW

File: client/src/components/Dashboard/index.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+          </StyledLink>
+        </DashboardDiv>
+
+        {/*card3*/}
+        <DashboardDiv>
+          <TitleDiv>
+            <Icon alt="cardlogo" src={cardLogo} />
+            <Title>Superpower Skills</Title>
+          </TitleDiv>
+          <Text>Find and harness your own superpowers’</Text>
```
Comment (FarahZaqout) [inline client/src/components/Dashboard/index.js]: remove the comma at the end of this line.

File: client/src/components/Dashboard/index.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+            to="/quiz"
+          >
+            <RestartImg src={restartVector} />
+            RESTART TEST
+          </StyledLink>
+          <StyledLink
+            background="#ED6237"
+            width="172px"
+            fontSize="20px"
+            Desktopmargin="-20rem 27rem 26rem"
```
Comment (FarahZaqout) [inline client/src/components/Dashboard/index.js]: prop name should start in lowercase.

File: client/src/components/Dashboard/DashboardStyledComponent.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+`
+
+const DashboardDiv = styled.div`
+  background: white;
+  border-radius: 21px;
+  margin: 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  max-height: 57vh;
```
Comment (FarahZaqout) [inline client/src/components/Dashboard/DashboardStyledComponent.js]: we shouldn't have max height in percentages.

File: client/src/components/Dashboard/DashboardStyledComponent.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  @media only screen and (min-width: 800px) {
+    max-height: 57vh;
```
Comment (FarahZaqout) [inline client/src/components/Dashboard/DashboardStyledComponent.js]: we shouldn't have max height in percentages.

File: client/src/components/Dashboard/DashboardStyledComponent.js [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+  display: flex;
+  flex-direction: column;
+  max-width: 800px;
+
+  @media only screen and (min-width: 800px) {
+    margin: auto;
+  }
+`
+
+const DashboardDiv = styled.div`
```
Comment (FarahZaqout) [inline client/src/components/Dashboard/DashboardStyledComponent.js]: we should try to make the names more descriptive.
```suggestion
const DashboardCard = styled.div`
```

**Comment by Nadeen123** [comment]
@FarahZaqout  resolved comments

**Pull request merged**